app/i18n/messages/fi.json

```diff
@@ -354,6 +354,7 @@
   "ResourceHeader.purpose": "Käyttötarkoitus",
   "ResourceHeader.capacity": "Kapasiteetti",
   "ResourceHeader.maxTime": "Enimmäisvarausaika",
+  "ResourceHeader.payable": "Maksullinen",
```

Comment:
```suggestion
  "ResourceHeader.payable": "Maksettava",
```

Reply:
This is the customer's request for now. We are still using the prices field but those can be empty even if the prices is not free of charge. (In that case customer wants to render 'Maksullinen` ).  The whole prices related stuff is changing soon. We are working on it at the moment, so both backend and frontend will change soon.